src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/acquireDotnetInstall.ts

```diff
@@ -0,0 +1,29 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+
+interface IDotnetAcquireResult {
+    dotnetPath: string;
+}
+
+export async function acquireDotnetInstall(outputChannel: vscode.OutputChannel): Promise<string> {
+    const version = "5.0.0";
```

Comment:
Can you store this version in the package.json so it's an easier update when we need to? For instance, this is how O# does it: 

https://github.com/OmniSharp/omnisharp-vscode/blob/136383252fb46c02fec693b6b06367d65cd1dfff/package.json#L31-L34

Hopefully we wont need to update our .NET Runtime version all that much 😄

Reply:
Considering it's a single property, I think the cost is equivalent between having it in the package.json or within the script. We get the benefit of not needing to import/read the JSON file to get the value here. 🤷🏽‍♀️

Reply:
Isn't it a single line?:
```TypeScript
const version = vscode.extensions.getExtension(razorExtensionId).packageJson.dotnetRuntimeVersion;
```

If the cost was equivalent wouldn't having it in the place where all versions/metadata information make more sense though?